Makefile

```diff
@@ -10,53 +10,60 @@ endif
 
 .PHONY: install
 install: ## make install # Install roles dependencies
-	@ansible-galaxy install --roles-path vendor/ -r requirements.yml
```

Comment:
You are right let's make the task decide whether to install it in `vendor` or not by checking both:

- `grep "roles_path =` ansible.cfg
- `[ -n ROLES_PATH ]`

Could you try to implement such a behavior?